_scss/_notes.scss

```diff
@@ -12,24 +12,45 @@ blockquote {
 
 blockquote > p:first-child {
   margin-top: 0;
+  font-weight: 700;
+  color: $info-color;
 }
 
-blockquote > p > strong {
-  color: $info-color;
+blockquote > p:first-child::before {
+  content: 'Note: ';
 }
 
 blockquote.warning {
   border-left-color: $warning-color;
 }
 
-blockquote.warning > p > strong {
+blockquote.warning > p:first-child {
   color: $warning-color;
 }
 
+blockquote.warning > p:first-child::before {
+  content: 'Be careful: ';
```

Comment:
I'm using the `warning` and `danger` classes, since other front-end frameworks like bootstrap use classes like that with the same semantics.